src/components/Bookmark/Bookmark.jsx

```diff
@@ -1,10 +1,19 @@
 import * as React from 'react'
 import './Bookmark.css'
 
-export default ({active}) => {
+export default ({ active }) => {
   return (
     <button className={active ? 'Bookmark active' : 'Bookmark'}>
```

Comment:
Since the only child of the button is an svg element, it would be nice to add an aria-label.
```suggestion
    <button className={active ? 'Bookmark active' : 'Bookmark'} aria-label="Bookmark card">
```